src/interactive.c

```diff
@@ -112,26 +106,22 @@ interactive_end(struct view *view)
 			wlr_output_layout_output_coords(view->server->output_layout,
 				view->output->wlr_output, &cursor_x, &cursor_y);
 
+			/*
+			 * Don't store natural geometry here (it was
+			 * stored already in interative_begin())
+			 */
 			if (cursor_x <= area->x + snap_range) {
-				view_snap_to_edge(view, "left");
+				view_snap_to_edge(view, "left", false);
 			} else if (cursor_x >= area->x + area->width - snap_range) {
-				view_snap_to_edge(view, "right");
+				view_snap_to_edge(view, "right", false);
```

Comment:
Inline comment about the new `false` would nice in this function as well, same for `view_maximize()` further down.

Reply:
Added inline comments.